api/envoy/config/route/v3/route.proto

```diff
@@ -119,6 +120,18 @@ message RouteConfiguration {
   //   is not subject to data plane buffering controls.
   //
   google.protobuf.UInt32Value max_direct_response_body_size_bytes = 11;
+
+  // [#not-implemented-hide:]
+  // A list of plugins and their configurations which may be used by a
+  // :ref:`envoy_v3_api_field_config.route.v3.RouteAction.cluster_specifier_plugin`
```

Comment:
I think this should be:

```
:ref:`cluster_specifier_plugin <envoy_v3_api_field_config.route.v3.RouteAction.cluster_specifier_plugin>`
```

Reply:
Looks like this actually works fine the way you have it, so nevermind. :)

Reply:
Discussed offline: we think this should be fine as-is, actually.  I can change it easily if needed, though.  (I already made the edits locally.)